I can't believe I spent hours debugging this stupid fucking thing. Just a
missing header file.


ld.lld-16: error: undefined symbol: partition_alloc::internal::InternalAllocator<char>::deallocate(char*, unsigned long)
>>> referenced by stats_collector.cc
>>>               allocator_core/stats_collector.o:(void partition_alloc::internal::StatsCollector::ReportTracesAndHistsImpl<(partition_alloc::internal::Context)0>(partition_alloc::StatsReporter&, partition_alloc::internal::StatsCollector::DeferredTraceEventMap<(partition_alloc::internal::Context)0> const&) const) in archive obj/base/allocator/partition_allocator/src/partition_alloc/liballocator_core.a


--- a/base/allocator/partition_allocator/src/partition_alloc/starscan/stats_collector.h
+++ b/base/allocator/partition_allocator/src/partition_alloc/starscan/stats_collector.h
@@ -14,7 +14,7 @@
 #include <unordered_map>
 #include <utility>
 
-#include "partition_alloc/internal_allocator_forward.h"
+#include "partition_alloc/internal_allocator.h"
 #include "partition_alloc/partition_alloc_base/threading/platform_thread.h"
 #include "partition_alloc/partition_alloc_base/time/time.h"
 #include "partition_alloc/partition_alloc_check.h"
